params/config.go

```diff
@@ -687,10 +687,12 @@ type ScrollConfig struct {
 
 // L1Config contains the l1 parameters needed to sync l1 contract events (e.g., l1 messages, commit/revert/finalize batches) in the sequencer
 type L1Config struct {
-	L1ChainId             uint64         `json:"l1ChainId,string,omitempty"`
-	L1MessageQueueAddress common.Address `json:"l1MessageQueueAddress,omitempty"`
-	NumL1MessagesPerBlock uint64         `json:"numL1MessagesPerBlock,string,omitempty"`
-	ScrollChainAddress    common.Address `json:"scrollChainAddress,omitempty"`
+	L1ChainId                       uint64         `json:"l1ChainId,string,omitempty"`
+	L1MessageQueueAddress           common.Address `json:"l1MessageQueueAddress,omitempty"`
+	L1MessageQueueV2Address         common.Address `json:"l1MessageQueueV2Address,omitempty"`         // TODO: set address once known
+	L1MessageQueueV2DeploymentBlock uint64         `json:"l1MessageQueueV2DeploymentBlock,omitempty"` // TODO: set block number once known
```

Comment:
Just noting inconsistency with the other deployment block param, which is passed through cli. We can consider moving that into genesis config later.

Reply:
you mean this one: https://github.com/scroll-tech/go-ethereum/blob/90d701cdce38d8c88b9180e42eedcfb06b8affcd/cmd/utils/flags.go#L1919

Agree. I think these kinds of network config parameter should be part of the genesis/rollup config